### PR TITLE
docs: replace headful with headed

### PR DESCRIPTION
**Pull request by mxschmitt**
There are far more occurrences of headful than for headed, so let's stick with headful and be consistent?

### CODE REVIEW

**Comment by pavelfeldman** [comment]
We are slowly going the other way around since headful has particular meaning not matching our needs.

**Comment by mxschmitt** [comment]
> We are slowly going the other way around since headful has particular meaning not matching our needs.

shall I do it the other way around or just close it? Cypress e.g. is only using the headed wording everywhere.

**Comment by pavelfeldman** [comment]
We can probably go headed everywhere. I hope it isn't too massive of a change.

File: docs/src/test-runners-python.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ To run your tests, use pytest CLI.
 # Run tests (Chromium and headless by default)
 $ pytest
 
-# Run tests in headful mode
-$ pytest --headful
+# Run tests in headed mode
+$ pytest --headed
```
Comment (mxschmitt) [inline docs/src/test-runners-python.md:33]: https://github.com/microsoft/playwright-pytest/pull/48

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -129,8 +129,8 @@ jobs:
     - run: node lib/cli/cli install-deps
     - run: bash packages/installation-tests/installation-tests.sh
 
-  headful_linux:
```
Comment (pavelfeldman) [inline .github/workflows/tests.yml:132]: Let's keep the original values for internal systems - easier to keep required bots assigned

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -166,10 +166,10 @@ it.skip('should work', async ({server, page}) => {
 });
 ```
 
-- To run tests in non-headless (headful) mode:
+- To run tests in non-headless (headed) mode:
 
 ```bash
-HEADFUL=1 npm run ctest
+HEADED=1 npm run ctest
```
Comment (pavelfeldman) [inline CONTRIBUTING.md:172]: ditto

File: src/server/browser.ts [hunk 4]
```diff
@@ -41,7 +41,7 @@ export type BrowserOptions = PlaywrightOptions & {
   isChromium: boolean,
   channel?: types.BrowserChannel,
   downloadsPath?: string,
-  headful?: boolean,
+  headed?: boolean,
```
Comment (pavelfeldman) [inline src/server/browser.ts:44]: Also, let's keep the code as is for simplicity.

**Review by pavelfeldman — COMMENTED** [review]
Let's only update the user-visible docs for now. Otherwise, there are too many moving parts!

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**